### PR TITLE
Add support to disable the end marker

### PR DESCRIPTION
**Pull request by ThiloAschebrock**
Since https://github.com/pierpo/react-archer/pull/134 it is now possible to add markers also to the start. To fully utilize this, see e.g. https://github.com/pierpo/react-archer/issues/77, it is important  to be able to hide the end marker. This is what this PR tries to achieve.

### CODE REVIEW

**Comment by pierpo** [comment]
Hi! Awesome, thanks for the contribution. It makes total sense and is consistent with the existing API :blush: 

I will manually test, then merge and publish soon :blush:

**Comment by ThiloAschebrock** [comment]
Thank you. I have updated README.md accordingly and added another example.

**Comment by ThiloAschebrock** [comment]
Any updates @pierpo?

**Comment by pierpo** [comment]
> Any updates @pierpo?

Sorry about the delay, I'll try to do it this week-end! Feel free to bump again if I don't 😄

File: example/App.js [hunk 1]
```diff
@@ -44,14 +47,11 @@ const App = () => {
...
-          <button onClick={() => setExampleId(2)}>Example 2</button>
-          <button onClick={() => setExampleId(3)}>Example 3</button>
-          <button onClick={() => setExampleId(4)}>Example 4</button>
-          <button onClick={() => setExampleId(5)}>Example 5</button>
-          <button onClick={() => setExampleId(6)}>Example 6</button>
-          <button onClick={() => setExampleId(7)}>Example 7</button>
-          <button onClick={() => setExampleId(8)}>Example 8</button>
+          {[...Array(9).keys()].map(value => (
+            <button key={value} onClick={() => setExampleId(value + 1)} style={{ marginRight: 8 }}>
+              `Example ${value + 1}`
```
Comment (pierpo) [inline example/App.js:52]: Oops 😉 

```suggestion
              {`Example ${value + 1}`}
```

It's fine I'll merge and fix on master!

**Review by pierpo — APPROVED** [review]

**Comment by pierpo** [comment]
Finally merged & published, thank you for the lovely contribution 😊

**Pull request merged**